kibana-reports/server/routes/utils/dataReportHelpers.ts

```diff
@@ -1,4 +1,3 @@
-import { async } from 'rxjs/internal/scheduler/async';
```

Comment:
Are the changes in this file related to the title of the PR? I feel this is a different optimization refactor

Reply:
That's just an unused import

Reply:
Not just this line, I'm referring to the other changes in the file as well- meant to highlight the whole file

Reply:
The other change is just rewriting the import to typescript style.